posthog/hogql/printer.py

```diff
@@ -1393,7 +1393,7 @@ def visit_call(self, node: ast.Call):
 
     def visit_placeholder(self, node: ast.Placeholder):
         if node.field is None:
-            raise QueryError("You can not use expressions inside placeholders")
+            raise QueryError("Unresolved placeholder")
```

Comment:
Is it possible to have a more descriptive error message here? All good if not, but this is what users will see on the sql editor